chk/modules/version/constants.py

```diff
@@ -13,14 +12,3 @@ class VersionStrToSpecConfigMapping:
     data: Dict = {
         "default:http:0.7.2": "chk.modules.http.entities.HttpSpec_V072",
     }
```

Comment:
this can be list, as we discussed. just to note.

Reply:
yes I though about that, I has other dependencies, I will eventually refactor those